homeassistant/components/ezviz/translations/fr.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Please remove all translations except the default English translation. The translations will be handled by Lokalise.

Reply:
Done.